app/models/compute_resource.rb

```diff
@@ -262,6 +262,15 @@ def set_console_password=(setpw)
     self.attrs[:setpw] = nil
   end
 
+  # this method is overwritten for Libvirt
```

Comment:
If this method is only used in libvirt, why do you include it here? You can also put the `attr_accessible` just in the Libvirt model